bjoern/statsd_tags.c

```diff
@@ -0,0 +1,32 @@
+#include <stdio.h>
+#include "statsd-client.h"
+
+#define MAX_MSG_LEN_WITH_TAGS 1000
+
+/* will change the original string */
+static void cleanup(char *stat)
```

Comment:
Why do we need this? Why don't we simply not put any of these chars in the strings in the source code?

Reply:
yeah, that makes sense. removed the function.